### ISSUE TITLE
Refresh should save metadata before downloading

### ISSUE THREAD

**Issue by obar1** [Owez/yark#116]
> it's possible to expose downloaded video asap

I did just tried to create a new dump of a channel and after hitting `yark refresh` all works
```
Loading sp01 channel..
Downloading metadata.. ◶
Parsing video metadata.. ◴
Parsing livestream metadata..
Parsing shorts metadata..
Downloading 154 new videos..
  • Downloaded ZSR9YEVMuY4        0.0%..
  • Downloaded 4qIjvuSl3Mw        0.0%..
```
**but** when I tried to open a new terminal session and I run `yark view`
the browser popup but no videos are showed 
```
sp01's videos
No videos found!
```

the req for enhacnment is that:
IT WOULD BE GREAT  if no too much of work to have 
**video downloaded -> video coverted -> video available** 1 by 1

now I think you do all in batch  :)

**Comment by obar1** [comment]
ofc this is an [enhancment ](https://github.com/Owez/yark/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement)

**Comment by Owez** [comment]
This is actually a feature on the master branch already, but I’ve stopped work on it for the upcoming rewrite unfortunately
For now you can do:
1. yark refresh foobar —skip-download 
2. yark refresh foobar —skip-metadata
(The double - has became — iPhone)

**Comment by obar1** [comment]
ok  - up to you to close this one or keep it open until it is fully implemented by pending work  you mentioned in master

**Comment by Owez** [comment]
I'll leave it open to make sure I include it in the rewrite if I'm implementing the CLI straight away haha

**Comment by Owez** [comment]
Closed by #140
